packages/vue/src/components/molecules/SfGallery/SfGallery.js

```diff
@@ -30,54 +30,94 @@ export default {
       type: Object,
       default() {
         return {
-          type: "slider",
+          type: 'slider',
           autoplay: false,
           rewind: false,
-          gap: 0
+          gap: 0,
         };
-      }
+      },
+    },
+    /**
+     * Image zoom inside or overlap the stage
+     */
+    inside: {
```

Comment:
imo `inside` is not sufficient descriptive name for this prop, wdyt about `insideZoom` or something like this?
and i think would also be to good to have a prop to disable zoom at all, wdyt?

Reply:
Nice ```insideZoom```. it's much more specific.
 I also think that the user doesn't want to have the effect of zoom in the image, WDYT prop called ```toggleZoom``` and by default it was ```true```?

Reply:
sounds great!